src/components/AddBook.jsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Good job @MasumaJaffery You have done a great job removing the `eslint-disable` comments as the previous reviewer mentioned. However, I noticed that there is one more left that hasn't been removed. So kindly remove this as well :wink: